Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -251,6 +255,46 @@ protected virtual IHostBuilder CreateHostBuilder()
             return builder;
         }
 
+        #if NET8_0_OR_GREATER
+        /// <summary>
+        /// Return one or more <see cref="HttpRequestMessage"/>s that will be used to invoke
+        /// Routes in your lambda function in order to initialize the asp.net and lambda pipelines
+        /// during <see cref="SnapshotRestore.RegisterBeforeSnapshot"/>,
+        /// improving the performance gains offered by SnapStart.
+        /// <para />
+        /// The returned <see cref="HttpRequestMessage"/>s must have a relative
+        /// <see cref="HttpRequestMessage.RequestUri"/>.
+        /// <para />.
+        /// Be aware that this will invoke your applications function handler code
+        /// multiple times.  Additionally, it uses a mock <see cref="ILambdaContext"/>
```

Comment:
Wordsmithing "Additionally, it uses a mock <see cref="ILambdaContext"/> which may not be fully populated." => "When the function handler is code as part of snapstart warm up the <see cref="ILambdaContext"/> instance will be a mock version which will not be fully populated."

Reply:
adjusted with minor tweak